include/glow/Graph/Graph.h

```diff
@@ -224,16 +230,18 @@ class Function final : public Named {
   Module *parent_;
 
   /// The log context associated with this function.
-  LogContext logCtx_;
+  LogContext *logCtx_ = nullptr;
```

Comment:
What's the motivation of changing this to a pointer? Seems like the Function always owns it (new/delete contained in constructor/destructor).